### PR TITLE
node-free

### LINKED ISSUE #16: Failure to compile on TypeScript 1.8
Currently ThermionSPA fails to compile because TS compiler behavior changed 
Following code snippet demonstrate 
https://gist.github.com/dmitry-a-morozov/47ce5ca63541085258f5
It fails with
Severity	Code	Description	Project	File	Line	Suppression State
Error	TS2322	Type 'string[]' is not assignable to type 'string'.	TypeGuardsFailure
on line 10

### LINKED ISSUE #17: remove NuGet dependency of jQuery.TypeScript.DefinitelyTyped
https://www.nuget.org/packages/Gluon.Client/1.0.2
image

That dependency should be removed. TypeScript users may choose to manage their TypeScript dependencies outside of NuGet, such as using [typings](https://www.npmjs.com/package/typings).

### PR DESCRIPTION
**Pull request by t0yv0**
* Fixes #17 
* Fixes #16 

Removes Node/NPM, moves TypeScript projects to VS code and tsconfig.json (they now build pretty simply via MSBuild using Paket-managed TSC, and via vanilla system tsc such as in VS code), and makes samlpes build on CI.

### CODE REVIEW

**Comment by t0yv0** [comment]
CC @dmitry-a-morozov

File: src/Gluon.Client/Gluon.Client.proj [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Minimal Build File to use NuGet-distributed TypeScript Compiler.
```
Comment (ctaggart) [inline src/Gluon.Client/Gluon.Client.proj]: Cool. Thanks for the example.

File: src/Gluon.Client/paket.template [hunk 2]
```diff
@@ -12,7 +12,5 @@ projectUrl http://www.tachyus.com/gluon/
 files
     dist/Gluon.d.ts ==> content\Scripts\typings
     dist/Gluon.js ==> content\Scripts
+    dist/Gluon.js.map ==> content\Scripts
     dist/Gluon.min.js ==> content\Scripts
-    dist/Gluon.min.js.map ==> content\Scripts
-dependencies
-    jQuery.TypeScript.DefinitelyTyped >= 2.2.1
```
Comment (ctaggart) [inline src/Gluon.Client/paket.template]: :+1:

File: src/Gluon.Client/paket.template [hunk 3]
```diff
@@ -12,7 +12,5 @@ projectUrl http://www.tachyus.com/gluon/
 files
     dist/Gluon.d.ts ==> content\Scripts\typings
     dist/Gluon.js ==> content\Scripts
+    dist/Gluon.js.map ==> content\Scripts
```
Comment (ctaggart) [inline src/Gluon.Client/paket.template]: :+1:

**Comment by ctaggart** [comment]
When we release the next version of Gluon, we may wish to bump up the version to `1.1.0` or something since this ends up putting a dependency on TypeScript 1.8. @panesofglass @t0yv0

**Comment by t0yv0** [comment]
+1.

**Pull request merged**